zabbix-probes/zabbix-probes-common/src/main/java/com/indigo/zabbix/utils/KeystoneTokenProvider.java

```diff
@@ -16,6 +16,10 @@
   @RequestLine("GET /v3/OS-FEDERATION/identity_providers/indigo-dc/protocols/oidc/auth")
   @Headers("Authorization: Bearer {token}")
   Response getKeystoneToken(@Param("token") String accessToken);
+  
+  @RequestLine("GET /v3/OS-FEDERATION/identity_providers/indigo-dc/protocols/iamoidc/auth")
```

Comment:
Are you getting that parameter from a property file? I thought to proceed that way bu then I changed my mind and I added an API. Can you show me your change?  I want to do exactly the same thing to reduce the conflicts